### DISCUSSION TITLE
Editing and Removing markers

### DISCUSSION THREAD

**Discussion by ghost** [olive-editor/olive#1890] [Q&A]
First, Thank you for creating this great video editor!

Second, I have tried to (re)move some markers I have put in the timeline of a project, but could not edit or remove them when pressed "M" on the keyboard. Could you help me with this, please?

Version: 0.2 (on the latest commit) for 64bit on Windows.

Thanks in advance!

**Comment by kburchfiel** [comment]
I don't know whether this was the case when you posted this question, but the markers can be deleted by either left clicking on them and pressing the Delete key or right clicking on them and choosing 'Delete' within the list of options that appears. I'm also using Olive 0.2 on a 64-bit version of Windows.

![image](https://github.com/olive-editor/olive/assets/32693822/b956077a-07bf-4cf2-8d0e-e470f2af73cc)
